api/v1alpha1/gatekeeper_types.go

```diff
@@ -127,9 +127,14 @@ type WebhookConfig struct {
 	// +optional
 	Resources *corev1.ResourceRequirements `json:"resources,omitempty"`
 	// +optional
+	Operations *[]OperationType `json:"operations,omitempty"`
```

Comment:
Why not just use `[]admregv1.OperationType`?

Also, it'd be best to avoid a pointer to a slice. It's just messy and I think your `== nil` check will still work.

Reply:
oh Because I want to add `+kubebuilder:validation:Enum:` on []admregv1.OperationType

Reply:
I changed it!!